integration_tests/2pc_test.go

```diff
@@ -902,11 +902,23 @@ func (s *testCommitterSuite) TestPessimisticLockIfExists() {
 	s.Equal(txn.GetLcokedCount(), 3)
 	s.Nil(txn.Commit(context.Background()))
 
+	// Primary key is not selected, but here is only one key.
```

Comment:
I think with this condition, there should be two test cases depending on whether the key exists.

When the key does not exist, we should ensure the primary remains `nil` after `LockKeys`.